test.js

```diff
@@ -100,24 +101,35 @@ test('support sync function with plan', function(t) {
   t.equal(result, 42);
 });
 
-test('support arrows with plan', t => {
```

Comment:
While I agree that arrow fn tests are probably useless, It could be useful a sync test with plan call.
Could you re-add one?